### PR TITLE
bring in prettierrc and remove overlapping eslint configs, so now prettier will solve most of the errors on its own

### PR DESCRIPTION
**Pull request by mcgnly**
this was driving me nuts- please feel free to comment and we can come to a joint conclusion about what the defaults SHOULD be, but they were conflicting as they were, and it wasn't easy for an autoformatter to do its job.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -3,15 +3,16 @@
...
-    "prettier",
-    "prettier/react",
     "plugin:import/errors",
     "plugin:import/warnings",
     "plugin:import/typescript"
   ],
-  "plugins": ["jsx-a11y", "prettier"],
+  "plugins": ["jsx-a11y"],
   "rules": {
-    "semi": 0,
```
Comment (DurkoMatko) [inline .eslintrc:14]: I think we should keep `"semi": 0,` in or? If not, we'll get ton of complains about missing semicolons everywhere I think.

File: .eslintrc [hunk 2]
```diff
@@ -3,15 +3,16 @@
...
+    // driving me nuts and would have involved changing a zillion files for no real benefit
+    "react/jsx-one-expression-per-line": "off",
+    "quotes": "off",
+    "no-unused-expressions": "off",
```
Comment (DurkoMatko) [inline .eslintrc:16]: agree

**Review by DurkoMatko — COMMENTED** [review]
Except for the semicolor rule, this looks fine to me :+1:

File: .eslintrc (hunk 1 above)
Comment (mcgnly) [reply .eslintrc:14]: for me it didn't change anything one way or the other, so 🤷 
happy to keep it in though

**Pull request merged**